common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
@cahrens, do you know what's going on here? Is there a different file that @waheedahmed should be editing, instead?